### ISSUE TITLE
"Bot Connected" loop

### ISSUE THREAD

**Issue by rlku** [nmynarcik/twitch-bot#3]
When ran the bot gets stuck in a loop where it just keeps saying "Bot Connected"

**Comment by graycatfromspace** [comment]
+1

**Comment by FailedSleep** [comment]
You need to make sure your botName and password (oath:) match completely, so if you generated your oath: for twitch user "SomeUser", the botName has to exactly be "SomeUser", no difference in case, or spaces.

**Comment by nmynarcik** [comment]
yeah I stopped working on this project so there might have been some changes from Slack and how their API works now.
